packages/playground-examples/copy/ja/TypeScript/Primitives/Unknown and Never.ts

```diff
@@ -1,14 +1,14 @@
-// Unknown
+// unknown型
 
-// Unknown is one of those types that once it clicks, you
-// can find quite a lot of uses for it. It acts like a sibling
-// to the any type. Where any allows for ambiguity - unknown
-// requires specifics.
+// unknown型は、非常に多くの箇所で使用されています。
+// unknown型は、any型とよく似た振る舞いをします。
+// 違いは、anyは曖昧なままでも使用できますが、 unknownは
+// 詳細を必要とする点です。
```

Comment:
こなれてないので、もうちょっとよくできそうですが、とりあえずsuggestしておきます
```suggestion
// anyは曖昧なままでも使用できますが、 unknownは
// 特定を必要とします。
```

Reply:
ありがとうございます、こんな感じでどうでしょうか？

```suggestion
// anyは曖昧なままでも使用できますが、 unknownは
// 型を特定する必要がある点が異なります。
```

Reply:
👍

Reply:
👍